frontend/templates/read.html

```diff
@@ -0,0 +1,123 @@
+{% load sass_tags %}
+
+<!DOCTYPE html>
+<head>
+    <title>Read {{ work.title }} online at unglue.it</title>
```

Comment:
I assume this head is mostly the same as on work page? If so, we'll want to extract the common code into one template.

Reply:
Unfortunately it is mostly different. the only common code is the meta tags, which we could extract away.

Reply:
that's a should do, then